src/entry.c

```diff
@@ -677,7 +677,24 @@ get_range(bitstr_t * bits, int low, int high, const char *names[],
 	if (state != R_FINISH || ch == EOF)
 		return (EOF);
 
-	for (i = num1; i <= num2; i += num3)
+	/* Make sure the step size makes any sense */
+	if (step > 1) {
+		if (high == high_ && low == low_) {
+			if (step > (high_ - low_)) {
+				fprintf(stderr, "Step size of '%i' will be ignored, since it exceeds the maximum possible step size of '%i' for the specified entry'\n", step, high_ - low_);
+			}
+		}
+		else { /* a range was specified */
+			if (high_ == low_) {
+				fprintf(stderr, "Step size of '%i' will be ignored for this entry, since it exceeds the maximum possible step size of '1' for the specifed range of '%i-%i'\n", step, low_, high_);
+			}
+			else if (step > (high_ - low_)) {
+				fprintf(stderr, "Step size of '%i' will be ignored for this entry, since it exceeds the maximum possible step size of '%i' for the specifed range of '%i-%i'\n", step, high_ - low_, low_, high_);
+			}
+		}
```

Comment:
I do not think we need special handling for these warnings. IMO the simple one above is sufficient.

Reply:
Alright, changed the code accordingly.

There is an edge-case when the min range equals the max range, e.g. `4-4/2`. For that, I forced `1` as the maximum step-size instead of 0. (Not that it would have any relevance, just thought a max step-size of 0 looks a bit odd :slightly_smiling_face: )

If you don't like it, we can as well just have `Warning: Step size 3 higher than possible maximum of 0`. I attached a testcase `example-crontab-invalid3` for that to the zip file.